vips/vip-285/bscmainnet.ts

```diff
@@ -0,0 +1,89 @@
+
+const VUSDT_IR = "0x9e8fbACBfbD811Fc561af3Af7df8e38dEd4c52F3";
+const VUSDC_IR = "0x9e8fbACBfbD811Fc561af3Af7df8e38dEd4c52F3";
+const VDAI_IR = "0x9e8fbACBfbD811Fc561af3Af7df8e38dEd4c52F3";
+const VFDUSD_IR = "0x958F4C84d3ad523Fa9936Dc465A123C7AD43D69B";
+const VTUSD_IR = "0x9e8fbACBfbD811Fc561af3Af7df8e38dEd4c52F3";
+const VUSDT_StableCoin_IR = "0x2ba0F45f7368d2A56d0c9e5a29af363987BE1d02";
+const VUSDT_GameFi_IR = "0x5ECa0FBBc5e7bf49dbFb1953a92784F8e4248eF6";
+const VUSDT_DeFi_IR = "0x5ECa0FBBc5e7bf49dbFb1953a92784F8e4248eF6";
+
+export const vip285 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-285 Chaos lab recommendation for the week 4th April(1/3)",
+    description: `This VIP will perform the following Risk Parameter action as per Chaos lab recommendation[Risk Parameter Updates 04/04/2024](https://community.venus.io/t/chaos-labs-stablecoin-ir-updates-04-04-24/4241). 
+    - USDT (Core pool) : update multiplier from 0.0687 to 0.1.
+    - USDC (Core pool) : update multiplier from 0.0687 to 0.1.
+    - DAI (Core pool) : update multiplier from 0.0687 to 0.1.
+    - TUSD (Core pool) : update multiplier from 0.0687 to 0.1.
+    - FDUSD (Core pool) : update multiplier from 0.0687 to 0.1 and jump multiplier from 2.5 to 5.
+    - USDT (StableCoin pool) : update multiplier from 0.0687 to 0.1.
+    - USDT (GameFi pool) : update multiplier from 0.10 to 0.15.
+    - USDT (DeFi pool) : update multiplier from 0.10 to 0.15. 
+      `,
+
+    forDescription: "I agree that Venus Protocol should proceed with the Risk Parameters Update's",
+    againstDescription: "I do not think that Venus Protocol should proceed with the Risk Parameters Update's",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds with the Risk Parameters Update's or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: VUSDT,
+        signature: "_setInterestRateModel(address)",
+        params: [VUSDT_IR],
+      },
+      {
+        target: VUSDC,
+        signature: "_setInterestRateModel(address)",
+        params: [VUSDC_IR],
+      },
+      {
+        target: VDAI,
+        signature: "_setInterestRateModel(address)",
+        params: [VDAI_IR],
+      },
+      {
+        target: VFDUSD,
+        signature: "_setInterestRateModel(address)",
+        params: [VFDUSD_IR],
+      },
+      {
+        target: VTUSD,
+        signature: "_setInterestRateModel(address)",
+        params: [VTUSD_IR],
+      },
+
+      {
+        target: VUSDT_StableCoin,
+        signature: "setInterestRateModel(address)",
+        params: [VUSDT_StableCoin_IR],
+      },
+      {
+        target: VUSDT_GameFi,
+        signature: "setInterestRateModel(address)",
+        params: [VUSDT_GameFi_IR],
+      },
+      {
+        target: VUSDT_DeFi,
+        signature: "setInterestRateModel(address)",
+        params: [VUSDT_DeFi_IR],
+      },
+    ],
+    meta,
+    ProposalType.REGULAR,
```

Comment:
Let's do it fast-track

Reply:
0c10417f73da1cdbe2c2e0f4beee2e3b3f64dbfb